<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCompositeFoodUI</class>
 <widget class="QDialog" name="EditCompositeFoodUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add or Edit Composite Food</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Owner:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboOwner">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Amounts</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtWeight">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboWeightUnit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Volume:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtVolume">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboVolumeUnit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtQuantity">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Servings:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtServings">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="FoodSearchControl" name="fscSearch" native="true"/>
   </item>
   <item>
    <widget class="SelectFoodControl" name="sfcResultsList" native="true"/>
   </item>
   <item>
    <widget class="FoodTree" name="ftComponents" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAndAdd">
       <property name="text">
        <string>Save and Add Another Food</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAndClose">
       <property name="text">
        <string>Save and Close</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FoodSearchControl</class>
   <extends>QWidget</extends>
   <header>food_search_control.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectFoodControl</class>
   <extends>QWidget</extends>
   <header>select_food_control.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FoodTree</class>
   <extends>QWidget</extends>
   <header>food_tree.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
